GVFS/GVFS.Platform.Mac/MacGitInstallation.cs

```diff
@@ -28,5 +29,17 @@ public string GetInstalledGitBinPath()
 
             return null;
         }
+
+        public string GetGVFSHooksPath()
+        {
+            // TODO(Mac): Use 'which' to find GVFS.Hooks (like the Windows platform uses "where")

```

Comment:
Can we just move to `which` now? I don't think it'll be too much work and we're making the change to use `which` to find Git in another PR.

Reply:
As discussed `which` is not able to find `GVFS.Hooks.exe` today - 
```
$ which GVFS.Hooks.exe
-bash: type: GVFS.Hooks.exe: not found
```

This is because it does not exist in the standard search paths. We can discuss the installer options to enable `which` (may be install a link like we do already for `gvfs`, or update `$PATH`). This is not directly related to this PR (service porting and auto-mount on Mac). I have created an [issue](https://github.com/Microsoft/VFSForGit/issues/1028) to track this. We can do this in a separate PR.